server/bin/pbench-trampoline

```diff
@@ -14,7 +14,7 @@ else
     printf -- "${_basename}: pbench-trampoline needs python3, either directly or through SCL\n" >&2
     exit 1
 fi
-export PYTHONPATH=$(dirname ${_dirname})/lib
+export PYTHONPATH=$(dirname ${_dirname})/lib:$(dirname ${_dirname})/common/bin:$(dirname ${_dirname})/common/lib/configtools:$PYTHONPATH
```

Comment:
So maybe we don't need an enhanced PYTHONPATH in the unittests scripts and the base file: this takes care of it.

Reply:
ok